platform/authorized_api_guide.md

```diff
@@ -38,22 +81,255 @@ curl 'https://<domain>/auth/token' -H "Content-Type: application/json" --data '{
 }' 
 ```
 
-The example response is: 
+Response example:
 ```
-{"token_type":"Bearer","expires_in":86400,"access_token":"your_access_token"}
+{
+    "token_type":"Bearer",
+    "expires_in":86400,
+    "access_token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJwcG1zIiwiYXVkIjoiaHR0cHM6XC9cL3Rlc3RpbmcucGl3aWsucHJvXC9zZXR5LCJzdWIiOiJkNmNkZGMxMS1iZDA1L0aW5ncyIsImlhdCI6MTUzNzI3MDQ1OSwiZXhwIjoxNTM3MzU2ODUTRhYmUtYWIyZC02YjlhNjIxZmU0ZDciLCJvcmciOiJkZWZhdWx0In0.Nec2mYFRv6manzXjq0sHQxINZvu-fbDYT8AedVHBKYvu1F9hYKaFReY8rNgfsMANw2OX8-IKpTrQb1DyRkG4nxpIEbob528_lPd7roho5mtKlE8sfS9WZE1piYOwaNDySDEUwUowgj2xBiJqSODjxBI6qVhLkynGEEeNBVh-lrUmlcjpYqUc3saHvX72L-rqbIHa_1dzGarR-dcPyns-RpKjZEILzUSYOHdM09KDti-xsG-nbKHGdP8fVEEJPyupnAfJPOLHQg_j1c5IvJSvTKVF3j4_zo6Zw5g8YkaheT9Iwph5BGHFRneXatcmbwKI8JzSDFi6CinzI-okYKRPbg"
+}
 ```
 
-Piwik PRO API tokens use [jwt](https://jwt.io/) format.
+Field `access_token` contains your token which then will be used for all API calls.
+
+#### Create an app
 
-Now, you can use obtained `access_token` for communication with Piwik PRO API.
-Field `expires_in` stands for time (in seconds) for token TTL.
-As token is a Bearer type, it must be **included in all API calls** within header.
+Request example:
 
 ```
-Authorization: Bearer your-token-here
+POST /api/apps/v2
 ```
-  
-## Deleting API Credentials
 
-Once you want to revoke the possibility of generating API token using given `CLIENT ID` and `CLIENT SECRET`,
-go to to `My profile` and click `Delete` button on selected API credentials.
+```
+curl 'https://<domain>/api/apps/v2' -H "Authorization: Bearer <your_access_token>" -H "Content-Type: application/vnd.api+json" --data '{
+  "data": {
+    "attributes": {
+      "timezone": "UTC",
+      "name": "AppName",
+      "urls": [
+        "http://example.com"
+      ],
+      "currency": "USD"
+    },
+    "type": "ppms/app"
+  }
+}'
+
+```
+
+Note, that you have to replace: 
+* `<domain>` with your PPMS instance URL,
+* `<your_access_token>` with your generated access token 
+
+Response example:
+```
+{
+   "data":{
+      "type":"ppms\/app",
+      "id":"b30e538d-4b05-4a75-ae25-7eb565901f38",
+      "attributes":{
+         "name":"AppName",
+         "addedAt":"2018-09-13T12:16:30+00:00",
+         "urls":[
+            "http:\/\/example.com"
+         ],
+         "timezone":"UTC",
+         "currency":"USD",
+         "excludeUnknownUrls":false,
+         "keepUrlFragment":true,
+         "eCommerceTracking":false,
+         "siteSearchTracking":true,
+         "siteSearchQueryParams":[
+            "q",
+            "query",
+            "s",
+            "search",
+            "searchword",
+            "keyword"
+         ],
+         "siteSearchCategoryParams":[
+
+         ],
+         "delay":500,
+         "excludedIps":[
+
+         ],
+         "excludedUrlParams":[
+
+         ],
+         "excludedUserAgents":[
+
+         ],
+         "gdpr":true,
```

Comment:
Asking just to be sure: Those fields are always returned even when Consent Manager is not available?

Reply:
yes they are, but possibly that's ok

Reply:
yup - we'll come back to this problem after implementing proper licensing - modifying API response because of product-bar config seems wrong